designs/dragonphy_top/constraints/gen_constraints.py

```diff
@@ -283,12 +341,13 @@
 set_max_transition {0.05*time_scale} -clock_path [get_clock clk_tx_hr]
 set_max_transition {0.1*time_scale} -clock_path [get_clock clk_tx_qr]
 
-# Set transition time for high-speed signals monitored from iacore and itx
-# transition time is 10% of a 4 GHz period.
+# Set transition time for high-speed signals monitored from
+# iacore and itx.  transition time is 10% of a 4 GHz period.
 
-foreach x $adbg_clk_pins {{
-    set_max_transition {0.025*time_scale} $x
-}}
+# TODO: figure out how to set a transition constraint on a black-box pin
+# foreach x $adbg_clk_pins {{
```

Comment:
This constraint didn't work, so I removed it.  Looks like applying ``max_transition`` to a black-box pin is problematic.  We still need to figure out a way to apply these constraints, though.